alembic-autogenerate.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+#
+# How to use this script:
+# 1. Load the LSST environment and setup sdm_schemas and felis.
+#        source loadLSST.bash
+#        setup felis
+#        setup -r /path/to/sdm_schemas
+# 2. Set the SDM_SCHEMAS_DIR environment variable to point to your sdm_schemas
```

Comment:
This is done by the `setup -r` command above, so it's not necessary as a separate step.
(Oh, and if the already-installed `sdm_schemas` is adequate and not a local clone, then `setup sdm_schemas` is sufficient.)